src/hooks/use-url-routing.js

```diff
@@ -80,6 +80,19 @@ export default function useUrlRouting(options) {
     [countSlideElements, isSlideElementOutOfBounds, isSlideOutOfBounds]
   );
 
+  const goToSlide = React.useCallback(
```

Comment:
I'm a little hesitant in exposing this method. The `onHistoryChange` function will correct any boundary errors. But it does seem like a lot of power to lend to the rest of the app.